tests/ParserTest.elm

```diff
@@ -605,6 +605,85 @@ expr =
                   , "\"'\""
                   , Ok (Literal (String "'"))
                   )
+                , ( "double quote"
+                  , "\"\\\"\""
+                  , Ok (Literal (String "\""))
+                  )
+                , ( "escape n"
+                  , "\"\\n\""
+                  , Ok (Literal (String "\n"))
+                  )
+                , ( "escape r"
+                  , "\"\\r\""
+                  , Ok (Literal (String "\u{000D}"))
+                  )
+                , ( "escape t"
+                  , "\"\\t\""
+                  , Ok (Literal (String "\t"))
+                  )
+                , ( "emoji"
+                  , "\"😃\""
+                  , Ok (Literal (String "😃"))
+                  )
+                , ( "escaped unicode code point"
+                  , "\"\\u{1F648}\""
+                  , Ok (Literal (String "🙈"))
+                  )
+                , ( "combo of escapes and chars"
+                  , "\"\\u{1F648}\\n\\r\\t\\abc123\""
+                  , Ok (Literal (String "🙈\n\r\tabc123"))
+                  )
+                ]
+              )
+            , ( "literal multiline string"
+              , [ ( "empty"
+                  , "\"\"\"\"\"\""
+                  , Ok (Literal (String ""))
+                  )
+                , ( "one space"
+                  , "\"\"\" \"\"\""
+                  , Ok (Literal (String " "))
+                  )
+                , ( "newline"
+                  , "\"\"\"\n\"\"\""
+                  , Ok (Literal (String "\n"))
+                  )
+                , ( "two numbers"
+                  , "\"\"\"42\"\"\""
+                  , Ok (Literal (String "42"))
+                  )
+                , ( "single quote"
+                  , "\"\"\"'\"\"\""
+                  , Ok (Literal (String "'"))
+                  )
+                , ( "double quote"
+                  , "\"\"\"\"\"\"\"\""
```

Comment:
I worry this isn't a correct test - this doesn't work in official Elm REPL for example.
![x](https://user-images.githubusercontent.com/149425/60688115-863d2700-9eb3-11e9-9588-63c78b8d2244.png)

Reply:
You can try to change it for something like `""" " """` - that demonstrates the inner `"` doesn't need escapes.

Reply:
Interesting. `""""""""` works when running `make test` but not in the REPL.

However, `""" "" """` works in both

Reply:
`""""""""` probably works because there's an even number of `"` (8)
I'd expect `"""""""` (7) to not work in both

Reply:
Maybe this would be easier to grok?

```suggestion
                  , String.repeat 7 "\""
```

Reply:
Would be a good idea, but `"""""""` is not supposed to parse. I'm suggesting `""" " """`, at which point `tripleQuote """ " """` or `tripleQuote " \" "` is what we need

Reply:
Ah I see! Fair point.

Reply:
Fixed by https://github.com/elm-in-elm/compiler/pull/31/commits/4bdfbef1a61dfa8c3448add3941f432c38ea895e